src/firefly/java/edu/caltech/ipac/firefly/server/servlets/HttpServCommands.java

```diff
@@ -43,9 +50,34 @@ public void processRequest(HttpServletRequest req, HttpServletResponse res, SrvP
                 fileNameExt = "";
             }
 
-            res.setHeader("Content-Disposition", "attachment; filename=" + fileName + fileNameExt);
             SearchManager am = new SearchManager();
 
+            if (tableSave != null) {
+                JSONObject json = new JSONObject();
+                String rVal = "";
+                try {
+                    //File myTmpFile = File.createTempFile(request.getRequestId(), fileNameExt, QueryUtil.getTempDir(request));
+                    File myTmpFile = File.createTempFile("TableSave-", ".tbl", QueryUtil.getTempDir(request));
+                    String replacedPrefix = ServerContext.replaceWithPrefix(myTmpFile);
+                    FileOutputStream out= new FileOutputStream(myTmpFile);
+                    am.save(out, request, tblFormat);
+                    json.put("success", true);
+                    json.put("serverFile", replacedPrefix);
+                    out.close();
+
+                }
+                catch (Exception e) {
+                    json.put("success", false);
+                    json.put("error",  e.getMessage());
+                }
+                rVal = json.toJSONString();
+                res.setContentType("application/json");
```

Comment:
Really minor.
I would remove the above line:  `String rVal = "";`
and declare it here instead...
```
    String rVal = json.toJSONString();
```